xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -1470,6 +1470,13 @@ CDemuxStream* CDVDDemuxFFmpeg::AddStream(int streamIdx)
       {
         stream = new CDemuxStream();
         stream->type = STREAM_DATA;
+        // Video (mp4) from GoPro cameras can have a 'meta' track used for a file repair containing
+        //  'fdsc' data, this is also called the SOS track. Ignore it or we will glitch bad on startup.
+        if (pStream->codec->codec_tag == MKTAG('f','d','s','c'))
+        {
+          stream->SetDiscard(AVDISCARD_ALL);
+          pStream->discard = AVDISCARD_ALL;
+        }        
```

Comment:
There's already a similar pull-request open:
https://github.com/xbmc/xbmc/pull/12579

Reply:
check my comments on this PR

Reply:
Superceded by #13095